src/Switch/__stories__/__snapshots__/Switch.stories.storyshot

```diff
@@ -212,15 +212,15 @@ exports[`Storyshots Component API/Switch Internal Labels 1`] = `
         >
           <span
             aria-label="on"
-            className="fd-fd-switch__text"
+            className=""
           />
           <span
             className="fd-switch__handle"
             role="presentation"
           />
           <span
             aria-label="off"
-            className="fd-fd-switch__text"
```

Comment:
the class `fd-fd-switch__text`.  I'm not sure this is meant to be `fd-switch--text` either, to preserve this PR only being about css-modules I'm just planning on taking it out.